src/app/api/v1/realtime/collections.py

```diff
@@ -215,6 +219,12 @@ async def material_counts_tree(
     status_code=HTTP_200_OK,
     responses={HTTP_404_NOT_FOUND: {"description": "Collection not found"}},
     tags=["Statistics"],
+    description="""Returns the average ratio of non-empty properties.
+    For certain properties, e.g. `properties.cclom:title`, the ratio of
+    elements which miss this entry compared to the total number of entries is calculated.
+    A missing entry may be `properties.cclom:title = null`. Not all properties are considered here.
```

Comment:
If we know the set of considered properties, then I would list them here explicitly. 

If the set of considered properties is available programmatically we could even inject it in the description string:

```python
CONSIDERED_PROPERTIES={"properties.cclom:title",...}

@router.get( 
...
description = f""" ...
Considered properties: {CONSIDERED_PROPERTIES} ..."""
)
async def score() -> :
   ...
```

Reply:
Good point, I will work on listing them

Reply:
Really hacky and I added a TODO to refactor this together with the other parts of score Depending on what WLO wants today.